EDM4eicVersion.h.in

```diff
@@ -27,7 +27,7 @@
 #define EDM4EIC_VERSION_MAJOR @EDM4EIC_VERSION_MAJOR@
 #define EDM4EIC_VERSION_MINOR @EDM4EIC_VERSION_MINOR@
 #define EDM4EIC_VERSION_PATCH @EDM4EIC_VERSION_PATCH@
-#define edm4hep_VERSION EDM4EIC_VERSION(EDM4EIC_VERSION_MAJOR, EDM4EIC_VERSION_MINOR, EDM4EIC_VERSION_PATCH)
+#define edm4eic_VERSION EDM4EIC_VERSION(EDM4EIC_VERSION_MAJOR, EDM4EIC_VERSION_MINOR, EDM4EIC_VERSION_PATCH)
 
 /// The encoded version with which EDM4EIC has been built
 #define EDM4EIC_BUILD_VERSION EDM4EIC_VERSION(EDM4EIC_VERSION_MAJOR, EDM4EIC_VERSION_MINOR, EDM4EIC_VERSION_PATCH)
```

Comment:
Also why do we need this.

Reply:
But it's exactly the same definition as edm4eic_VERSION

Reply:
Yeah I don't disagree ;-)